### PR TITLE
Add UserLoginWithEmail mutation call in webapp

### PR DESCRIPTION
**Pull request by miltoneiji**
As stated, this pr implements the UserLoginWithEmailMutation that was implemented on the server but not being used by the webapp.
#15

### CODE REVIEW

**Review by jgcmarins — COMMENTED** [review]
nice
I would try to use formik + yup:
- https://github.com/jaredpalmer/formik/
- https://github.com/jquense/yup

**Comment by miltoneiji** [comment]
Thanks! I will try your recommendation!

File: webapp/src/screens/SignInScreen/SignInScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+const SignInFormSchema = yup.object().shape({
+  email: yup.string()
+    .email('Email is not valid!')
+    .required('Email is required!'),
+  password: yup.string()
+    .min(6, 'Password has to be longer than 6 characters!')
+    .required('Password is required!'),
+});
+
+export interface ISignInFormInput {
```
Comment (miltoneiji) [inline webapp/src/screens/SignInScreen/SignInScreen.tsx:19]: I think I should start to refactor this project and move all interfaces to a single file/directory 🤔 
It is becoming a mess.

**Comment by miltoneiji** [comment]
I liked yup + formik, that's very powerful, thanks!

File: webapp/src/screens/SignInScreen/SignInScreen.tsx (hunk 1 above)
Comment (jgcmarins) [reply webapp/src/screens/SignInScreen/SignInScreen.tsx:19]: I think it is a good practice to define types next to code

**Pull request closed without merging**